src/server/blockchain/index.js

```diff
@@ -0,0 +1,17 @@
+const Web3 = require('web3');
```

Comment:
Is it possible to use `import` syntax here (for constistency) or does some other part require this syntax?

Reply:
We have `web3` already defined here: https://github.com/lazaruslabs/culturestake/blob/master/src/common/services/web3.js is it possible to extend the needed functionality there?

Reply:
Yup, didn't realize this was here